console/src/main/java/org/georchestra/console/ws/backoffice/orgs/OrgsController.java

```diff
@@ -386,7 +375,7 @@ public void getAreaConfig(HttpServletResponse response) throws IOException, JSON
             LOG.info("Could not parse value", e);
         }
         JSONObject areas = new JSONObject();
-        areas.put("url", areasUrl);
+        areas.put("url", "/console/public/area.geojson");
```

Comment:
Doesn't this look like technical debt ?

Reply:
We have replaced the discovery of "area" by an HTTP endpoint which will send you a file or a redirect to another HTTP endpoint. The configuration is now part of this "Area HTTP endpoint". Before doing this I looked at the first part of the path  and if it can be customized. It seems not, since `/console` is hardcoded everywhere in the manager js code (and in some Java class file).

Reply:
Sorry I should have been more explicit.
"url: /console/public/area.geojson" does not need to be part of the areaConfig.json response, since it is now static.
You could have modified the JS code to directly hit /console/public/area.geojson instead...